src/components/MessagesList/MessagesGroup/Message/MessagePart/FilePreview.vue

```diff
@@ -213,20 +235,14 @@ export default {
 	image. */
 	display: inline-block;
 
-	/* Show a hover colour around the preview when navigating with the
-	 * keyboard through the file links (or hovering them with the mouse). */
-	&:hover,
-	&:focus,
-	&:active {
-		.preview {
-			background-color: var(--color-background-hover);
+	border-radius: 16px;
 
-			/* Trick to keep the same position while adding a padding to show
-			 * the background. */
-			box-sizing: content-box !important;
-			padding: 10px;
-			margin: -10px;
-		}
+	&:hover,
+	&:focus {
+		background-color: var(--color-background-hover);
+		/* Trick to keep the same position while adding a padding to show
+			* the background. */
+		box-sizing: content-box !important;
```

Comment:
While this looks good in the upload editor in my opinion the look in the messages list has regressed. Therefore I would suggest to have both the previous style and the new one, and use one or the other depending on whether the `upload-editor` class is set or not.

Reply:
What do you think has regressed in particular? Before there was an unneeded hover feedback on the preview itself that partially covered the sender's username.

Reply:
> there was an unneeded hover feedback on the preview itself

I think that the hover feedback should be there, as it shows the user that she can interact with the file. The problem is that the whole message is "highlighted" on hover when showing the reply button, and that hides the specific feedback for the file. Anyway, I would be fine with removing the hover feedback on the file due to that overlap (but I would prefer to find a better solution nevertheless).

> that partially covered the sender's username

[I know](https://github.com/nextcloud/spreed/pull/2778#issuecomment-598234022), it has been haunting me since I added it :-P

However, even if it covers the sender's username I prefer that small, focused frame on the file:
![Message-File-Focus-Before](https://user-images.githubusercontent.com/26858233/90001223-70428e00-dc91-11ea-90d8-6b6533f43069.png)

Rather than highlighting the whole message body (even if it actually matches the real clickable area):
![Message-File-Focus-After](https://user-images.githubusercontent.com/26858233/90001240-7a648c80-dc91-11ea-97b0-7296f8875fc8.png)

For me the previous version feels and looks much better when navigating with the keyboard. Also the change is not directly related to editing the upload, so in any case I would address it in a separate pull request too.

Reply:
So this is the current behavior (before this pr):
![Peek 2020-08-13 09-43](https://user-images.githubusercontent.com/26852655/90107674-75fbaa80-dd49-11ea-8a7d-1416d5701821.gif)

So you get the big hover feedback of the message but no clickability unless you hover the preview, and since the hover color is the same, you get the hover feedback of the preview only through unwanted stuff:
- overflow from the message on the left;
- invading the name on top;

The keyboard navigation is a bug with how messages hover feedback is implemented. For a reference of how it should really look refer to the mouse navigation feedback. Plus, I think that a bigger click area is generally better, since that space is unused anyway... I implemented the change in this pr because I'm adapting the filepreview component. 


![Peek 2020-08-13 09-31](https://user-images.githubusercontent.com/26852655/90106953-6af44a80-dd48-11ea-8c57-ae414d8bbc80.gif)